### PR TITLE
feature: bundle all modules

### PR DESCRIPTION
**Pull request by thetutlage**
Fixes: FRMW-2722

### CODE REVIEW

**Review by carlos-r-l-rodrigues — APPROVED** [review]

**Review by adrien2p — APPROVED** [review]

File: packages/core/modules-sdk/src/definitions.ts [hunk 1]
```diff
@@ -7,31 +7,32 @@ import {
...
-  [Modules.WORKFLOW_ENGINE]: "@medusajs/workflow-engine-inmemory",
-  [Modules.REGION]: "@medusajs/region",
-  [Modules.ORDER]: "@medusajs/order",
-  [Modules.API_KEY]: "@medusajs/api-key",
-  [Modules.STORE]: "@medusajs/store",
-  [Modules.CURRENCY]: "@medusajs/currency",
-  [Modules.FILE]: "@medusajs/file",
-  [Modules.NOTIFICATION]: "@medusajs/notification",
-  [Modules.INDEX]: "@medusajs/index",
+  [Modules.AUTH]: "@medusajs/medusa/modules/auth",
```
Comment (olivermrbl) [inline packages/core/modules-sdk/src/definitions.ts:10]: question: How would an end-user module configuration look like in `medusa-config.js`?

For example, if you want to specify auth or payment providers.

**Review by olivermrbl — APPROVED** [review]

**Pull request merged**